options.html

```diff
@@ -7,9 +7,25 @@
     <link rel="stylesheet" href="/dependencies/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="css/options.css" />
     <script src="/dependencies/bootstrap/dist/js/bootstrap.min.js"></script>
-    <script src="js/allTenants.js"></script>
-    <script src="js/tenants.js"></script>
-    <script src="js/options.js"></script>
+    <script type="module" src="background.js"></script>
+    <script type="module" src="js/allTenants.js"></script>
+    <script type="module" src="js/tenants.js"></script>
+    <script type="module" src="js/options.js"></script>
+<!--    <script type="module">-->
```

Comment:
The example from the docs is still not clear to me, but this was an attempt to get the modules/service worker approach working in the first instance.